app/till_app.rb

```diff
@@ -0,0 +1,43 @@
+require 'sinatra/base'
+require './app/lib/till'
+
+class TillApp < Sinatra::Base
+
+  set :views, Proc.new { File.join(root, "views") }
+  set :public_folder, Proc.new { File.join(root, "public") }
+
+  till = Till.new
```

Comment:
I know we did this in battleships, but this really isn't the way to go - Sinatra shouldn't be storing state like this, if you need to persist the till object across requests then you would use sessions or a database (which is the more standard way of handling this). This would not go down well in a real life tech test!